File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcObjectDefinition/IfcContext/IfcContext.cs
public class IfcContext : IfcObjectDefinition
{
    public IfcLabel ObjectType { get; set; }
    public IfcLabel LongName { get; set; }
    public IfcLabel Phase { get; set; }
    public SET [1:?] OF IfcRepresentationContext RepresentationContexts { get; set; }
    public IfcUnitAssignment UnitsInContext { get; set; }

    // === EXTENDED BY ===
    // IfcProject
    // IfcProjectLibrary

    // === INVERSE CLAUSES ===
    // IsDefinedBy : SET [0:?] OF IfcRelDefinesByProperties FOR RelatedObjects
    // Declares : SET [0:?] OF IfcRelDeclares FOR RelatingContext
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcObjectDefinition/IfcContext/IfcContext.vb
Public Class IfcContext Inherits IfcObjectDefinition
    Public ObjectType As IfcLabel
    Public LongName As IfcLabel
    Public Phase As IfcLabel
    Public RepresentationContexts As SET [1:?] OF IfcRepresentationContext
    Public UnitsInContext As IfcUnitAssignment

    ' === EXTENDED BY ===
    ' IfcProject
    ' IfcProjectLibrary

    ' === INVERSE CLAUSES ===
    ' IsDefinedBy : SET [0:?] OF IfcRelDefinesByProperties FOR RelatedObjects
    ' Declares : SET [0:?] OF IfcRelDeclares FOR RelatingContext
End Class
